app/controllers/donations_controller.rb

```diff
@@ -24,13 +24,15 @@ def create
     respond_to do |format|
       if @donation.save
         DonationMailer.thankyou(@donation).deliver_now
-        format.html { redirect_to root_url, 
-                      :flash => { :success => "Thank you for your generous donation!"}
-                    }
+        format.html {
+          redirect_to root_url,
+          flash: { success: "Thank you for your generous donation!"}
+        }
       else
-        format.html { redirect_to new_donation_path,
-                      :flash => { :error => "There was an error processing your donation, please retry."} 
-                    }
+        format.html {
+          redirect_to new_donation_path,
+          flash: { error: "There was an error processing your donation, please retry."}
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space inside } missing.